region.go

```diff
@@ -181,6 +181,16 @@ var Regions = map[string]Region{
 		SysTypes: []string{"s922", "e980"},
 		VPCZones: []string{"us-east-1", "us-east-2", "us-east-3"},
 	},
+	"che": {
+		Description: "Chennai, India",
+		VPCRegion:   "",
+		COSRegion:   "",
+		Zones: []string{
+			"che01",
+		},
+		SysTypes: []string{"s922", "e980"},
```

Comment:
Hope you have validated the sys types available in Chennai! If not you can check via PowerVS workspace in cloud.

Reply:
I have checked the machine types for this zone `che01 ` I can see only these two machine types 

Do we need to include `Dedicated host types` as well  in sysTypes ?